sys-apps/debianutils/debianutils-4.9.ebuild

```
@@ -24,15 +27,20 @@ src_configure() {
 src_install() {
 	into /
 	dobin tempfile run-parts
-	if use installkernel ; then
+	if use systemd-boot ; then
+		dodir /sbin
+		dosym ../usr/bin/kernel-install /sbin/installkernel
```

Maybe we could use the split-usr USE flag to control the symlink target here.

---

```suggestion
		dosym ../$(usex split-usr 'usr/' '')bin/kernel-install /sbin/installkernel
```

Something like this?

---

(or should it go to `/usr/sbin/installkernel` then?)

---

I think your proposed change should suffice.